docs/reference/commandline/search.md

```diff
@@ -19,6 +19,69 @@ parent = "smn_cli"
       --no-trunc=false     Don't truncate output
       -s, --stars=0        Only displays with at least x stars
 
+## Examples
+
+### Search images by keywords
```

Comment:
can you change "keywords" to "keyword"? The search function (afaik) only supports a single keyword when searching.